content/html/concepts/elements/terms/style/style.md

```diff
@@ -0,0 +1,80 @@
+---
+Title: '<style>'
+Description: 'This element is used to apply a simple style sheet to an HTML document'
+Subjects:
+ - 'Web Development'
+ - 'Web Design'
+Tags: 
+ - 'Tags'
+ CatalogContent:
+ - 'learn-html'
+ - 'paths/front-end-engineer-career-path'
+---
+The **`<style>`** element is used to apply a styles to an HTML document
+
+## Syntax
+
+```html
+<!-- In-document style -->
+<style>
+  // CSS element goes here
+</style>
+
+<!-- Inline style -->
+<h1 style="color:yellow;text-align:center">This is a header</h1>
+<p style="color:red">This is a paragraph.</p>
+
+<!-- External style sheet -->
+<head>
+  <link rel="stylesheet" href="styles.css">
+</head>
+```
+The `<style>` tag is used to define style information (CSS) for a document or an HTML page.
+
+Inside the `<style>` element you specify how HTML elements should render in a browser.
+
+The `<style>` element must be included inside the `<head>` section of the document. In general, it is better to put your styles in external stylesheets and apply them using `<link>` elements.
```

Comment:
```suggestion
The `<style>` element must be included inside the `<head>` section of the document. All CSS styles follow the same [anatomy](https://www.codecademy.com/resources/docs/css/anatomy) for rulesets and declarations like external stylesheets.

The following attributes are available to the `<style>` element:
```

Reply:
Add a spacing after the code block and this paragraph